app/ibc.go

```diff
@@ -1,207 +0,0 @@
-package app
```

Comment:
How can we support IBC without this file?

Reply:
ah, I missed you had initiated some of the IBC stuff. we will need to add some stuff back in. I'd prefer to do that as part of the bridging work so we have a clear IBC integration we are working towards to better understand what code we are generating.

Reply:
Makes sense, it was generated anyways so we can regenerate it when we are ready.